### PR TITLE
Recompile configs (3.10)

### PR DESCRIPTION
**Pull request by dianabarsan**
# Description

Recompiles `standard` and `default` configs using latest `medic-conf@3.4.1`.

medic/cht-core#7003

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by dianabarsan** [comment]
@newtewt would you mind having a look at this? thank so much!

**Comment by dianabarsan** [comment]
Oh nevermind, there's a test that compares the branch app_settings with what medic-conf generates, and it's failing. So I guess this is blocked until I release medic-conf. Sorry for the trouble, @newtewt .

File: config/standard/test/check-app-settings-is-committed [hunk 1]
```diff
@@ -15,7 +15,7 @@ EOF
 trap explain_error ERR
 
 echo '[check-app-settings-js-is-committed] Generating app_settings.json...'
-./node_modules/.bin/medic-conf --no-check --local compile-app-settings
```
Comment (dianabarsan) [inline config/standard/test/check-app-settings-is-committed:18]: `--no-check` was ninja-removed with https://github.com/medic/medic-conf/pull/175

**Comment by dianabarsan** [comment]
@newtewt I think this should be good. I installed both configs, and everything seems to work, with one known issue: https://github.com/medic/cht-core/issues/7009

Would you mind having a look ?

File: config/default/test/check-app-settings-is-committed [hunk 2]
```diff
@@ -15,7 +15,7 @@ EOF
 trap explain_error ERR
 
 echo '[check-app-settings-js-is-committed] Generating app_settings.json...'
-./node_modules/.bin/medic-conf --no-check compile-app-settings
+./node_modules/.bin/medic-conf --no-check --local compile-app-settings
```
Comment (newtewt) [inline config/default/test/check-app-settings-is-committed]: I think medic-conf should work without a check. I'll open a separate issue for that. The `--no-check` flag I believe doesn't do anything anymore. It became  `--check` when we switched to a library to parse args.

File: config/default/app_settings.json [hunk 3]
```diff
@@ -343,9 +343,9 @@
...
...'value'),s(e,t,'translate'),s(e,t,'filter'),s(e,t,'width'),s(e,t,'icon'),e.context&&(t.context={},s(e.context,t.context,'count'),s(e.context,t.context,'total')),t}));return e.modifyContext&&e.modifyContext(t,n),{label:e.label,fields:i}}function s(e,t,r){switch(typeof e[r]){case'undefined':return;case'function':t[r]=e[r](n);break;default:t[r]=e[r]}}}e.exports=function(e,t,r){var s=e.fields||[],a=e.context||{},o=e.cards||[],l=t&&('contact'===t.type?t.contact_type:t.type),u={cards:[],fields:s.filter((function(e){var t=n(e.appliesToType),r=t.filter((function(e){return e&&'!'===e.charAt(0)}));if((0===t.length||t.includes(l)||r.length>0&&!r.includes('!'+l))&&(!e.appliesIf||e.appliesIf()))return delete e.appliesToType,delete e.appliesIf,!0}))};return o.forEach((function(e){var t,s,o,d,c=n(e.appliesToType);if(c.includes('report')&&c.length>1)throw new Error('You cannot set appliesToType to an array which includes the type \\'report\\' and another type.');if(c.includes('report'))for(t=0;t<r.length;++t)s=r[t],!(d=s)||d.errors&&d.errors.length||(o=i(e,a,s))&&u.cards.push(o);else{if(!c.includes(l)&&c.length>0)return;(o=i(e,a))&&u.cards.push(o)}})),u.context=a,u}}])})); return ContactSummary;",
```
Comment (newtewt) [inline config/default/app_settings.json:346]: This block is hard to verify in code review. My only real check is that there's a decent amount less code in the rules section now. Does that seem right to you?

**Review by newtewt — CHANGES_REQUESTED** [review]
I have some questions before approving but otherwise should be a quick turnaround.

File: config/default/test/check-app-settings-is-committed (hunk 2 above)
Comment (dianabarsan) [reply config/default/test/check-app-settings-is-committed]: I thiiink this code has already changes and you've reviewed an old version.

Comment (dianabarsan) [reply config/default/test/check-app-settings-is-committed]: I even added a comment above removing `--no-check` :D

Comment (newtewt) [reply config/default/test/check-app-settings-is-committed]: yeah I see it now.  I believe I got there by clicking the link from my email. A bit odd.

File: config/default/app_settings.json (hunk 3 above)
Comment (dianabarsan) [reply config/default/app_settings.json:346]: I'm having another look. 

I added the `check-app-settings-is-committed` to the testing scheme for `default` as well (it was only running for `standard`), so the test says it's ok :D 
https://travis-ci.com/github/medic/cht-core/jobs/491376417#L14844

Comment (dianabarsan) [reply config/default/app_settings.json:346]: Ok, I checked the two sources and compared char count: 

`contact_summary`: 
- old: 69679 chars
- new: 69614 chars
- diff: -65 chars 

`rules`:
- old: 18289 chars
- new: 18321 chars
- diff: -32 chars

I think the visual differences are caused by wrapping.

**Review by newtewt — APPROVED** [review]

File: config/default/app_settings.json (hunk 3 above)
Comment (newtewt) [reply config/default/app_settings.json:346]: 👍

**Pull request merged**